src/System.Data.Odbc/tests/System.Data.Odbc.Tests.builds

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't need the builds file. You should probably cherry-pick the commit I sent you offline (https://github.com/weshaggard/corefx/commit/c4a91f39e8ab1403baf1cd5ee873509230d7f5d2) to fix up the projects and solution.